include/crm/common/iso8601_internal.h

```diff
@@ -27,6 +27,7 @@ char *pcmk__time_format_hr(const char *format, const pcmk__time_hr_t *hr_dt);
 const char *pcmk__epoch2str(const time_t *when);
 const char *pcmk__readable_interval(guint interval_ms);
 crm_time_t *pcmk__copy_timet(time_t source);
+char *pcmk__now_as_string(uint32_t flags);
```

Comment:
We do have pcmk__epoch2str(), if you can live with its format

Reply:
I don't have a strong preference on format, was just trying to keep the existing one. `pcmk__epoch2str()` is currently inflexible and doesn't show time zone.

I'm open to adding a flags argument and somehow telling it to use `ctime()` under some condition (a particular flag or a boolean argument).

Reply:
Yeah, that sounds reasonable. It gets confusing when there are multiple functions for similar functionality. The condition isn't too important, it could be flags==0, or crm_time_log_with_timezone not set, or whatever.